Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -1283,6 +1283,7 @@ def addKalmanTracks(
     clusters: str = None,
     calibrator: acts.examples.MeasurementCalibrator = acts.examples.makePassThroughCalibrator(),
     logLevel: Optional[acts.logging.Level] = None,
+    whiteboardAliasTracks: str = "tracks",
```

Comment:
why do we need this?

Reply:
Ah right, I didn't realized that we can reassign alieses, so it should be possible to first run+write the first fitter, and then run+write the second fitter.
Can you try that @AJPfleger? If this works, this change is not needed I think.

Reply:
yes, I am waiting for the physmon results. it worked locally :)